netbox/core/api/views.py

```diff
@@ -199,7 +215,11 @@ def get_task_from_id(self, task_id):
 
         return task
 
-    @extend_schema(responses={200: OpenApiTypes.OBJECT})
+    @extend_schema(
+        operation_id='core_background_tasks_retrieve_by_id',
+        parameters=[OpenApiParameter(name='pk', type=OpenApiTypes.STR, location=OpenApiParameter.PATH)],
```

Comment:
Is this redundant to line 192 above? (Also, here we call the parameter `pk` but above it's named `id`.)